crates/cxx-qt-gen/src/parser/mod.rs

```diff
@@ -45,11 +47,36 @@ fn check_safety(method: &ForeignItemFn, safety: &Safety) -> Result<()> {
 ///
 /// Note: This modifies the method by removing those doc attributes
 pub fn separate_docs(method: &mut ForeignItemFn) -> Vec<Attribute> {
-    let mut docs = vec![];
-    while let Some(doc) = attribute_take_path(&mut method.attrs, &["doc"]) {
-        docs.push(doc);
+    method
+        .attrs
+        .iter()
+        .filter(|attr| path_compare_str(attr.meta.path(), &["doc"]))
+        .cloned()
+        .collect()
+}
+
+pub fn parse_attributes<'a>(
```

Comment:
doc line would be good, and as described in another comment would be good to get the word `require` into the name so that it matches the syn API. eg `parse_require_attributes`

Reply:
Done